pkg/disttask/framework/storage/task_table.go

```diff
@@ -844,6 +844,22 @@ func (mgr *TaskManager) GetSubtasksWithHistory(ctx context.Context, taskID int64
 	return subtasks, nil
 }
 
+// GetAllTasks gets all tasks with basic columns.
+func (mgr *TaskManager) GetAllTasks(ctx context.Context) ([]*proto.TaskBase, error) {
```

Comment:
maybe skip `pending`, it might be too many of them for `import into`

Reply:
But will it lose the original purpose of the metrics?